python/kfserving/README.md

```diff
@@ -1 +1,39 @@
-TODO
\ No newline at end of file
+# KFServing
+
+KFServing is a unit of model serving, and every model type (XGBosst, PyTorch, Tensorflow etc.) extends KFServing. At a highlevel it defines a model, a server and storage mechanism for models.
+
+A model has essentially three functions:
+
+* Pre processing
+* Prediction 
+* Post processing
+
+A server on the other hand provides many funtionalities, inclding among others
+
+* Registering a model and starting the server
+* Prediction Handler
+* Liveness Handler 
+* Metrics Handler 
+
+Models to load in KFServing can come from following three types of storage locations currently
```

Comment:
KFService supports the following storage providers

Reply:
Done